executable/microsoft_pe.ksy

```diff
@@ -408,3 +413,134 @@ types:
         size: length - 8
         doc: Contains a certificate, such as an Authenticode signature.
     doc-ref: 'https://docs.microsoft.com/en-us/windows/desktop/debug/pe-format#the-attribute-certificate-table-image-only'
+
+  resource_directory_table:
+    params:
+      - id: depth
+        type: u4
+      - id: section_file_offset
+        type: u4
+      - id: section_virtual_address
+        type: u4
+    seq:
+      # This field is reserved for future use. It is currently set to zero.
+      - id: characteristics
+        type: u4
+      - id: time_date_stamp
+        type: u4
+        -orig-id: 'TimeDateStamp'
```

Comment:
Nit: if nothing else is enclosed in quotes, why should be this one?

```suggestion
        -orig-id: TimeDateStamp
```

Reply:
Agreed